H/H3/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "H3"
+version = v"3.7.2"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/uber/h3/archive/v3.7.2.zip", "805f787912608b09afa7c0e9c432c0e4b43f17a9d10faed880b3f74f038ce576")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/h3-3.7.2/
+cat <<EOF > CMakeLists.txt.patch
+diff -uNr h3-3.7.2-original/CMakeLists.txt h3-3.7.2/CMakeLists.txt > CMakeLists.txt.patch
+--- h3-3.7.2-original/CMakeLists.txt    2021-08-12 16:32:04.000000000 +0900
++++ h3-3.7.2/CMakeLists.txt 2021-08-12 16:33:03.000000000 +0900
+@@ -58,6 +58,9 @@
+
+ project(h3 LANGUAGES C VERSION \${H3_VERSION})
+
++set(CMAKE_C_FLAGS "-std=c99 \${CMAKE_C_FLAGS}")
++set(CMAKE_C_LINK_FLAGS "-lm \${CMAKE_C_LINK_FLAGS}")
++
+ set(H3_COMPILE_FLAGS "")
+ set(H3_LINK_FLAGS "")
+ if(NOT WIN32)
+EOF
+patch -p1 -i CMakeLists.txt.patch
```

Comment:
Wouldn't
```suggestion
export CLFAGS="-std=c99"
export LDFLAGS="-lm"
```
work just fine?

Reply:
it doesn't  worked in my local docker container. I commented fixed build script with `-D` option.